src/core/core.h

```diff
@@ -44,8 +44,12 @@ __extension__({				\
 #define CACHE_LINE_SIZE 64
 #endif
 
-#define INIT 0
-#define DEINIT UINT_MAX
+enum rootsim_event {
```

Comment:
If we share definitions between headers, which is fine, I think this should at least be documented in a comment.

Reply:
That's right. Actually, I think ROOT-Sim.h should be generated from annotations in the code.

Reply:
But you don't like annotations! :smile: